TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GMR/Robot/Robot.java

```diff
@@ -75,11 +77,13 @@ public Robot (HardwareMap hardwareMap, Telemetry telemetry) {
         relicGrab = new RelicGrab(relicLift, slideLift, relicTilt, relicClamp);
 
         blockLift.clamp(false, true, false, false);
+
+        latchLift = new LatchLift(liftMotor, telemetry);
     }
 
-    public void setServos() {
+    /*public void setServos() {
         rightColor.setPosition(0);
         leftColor.setPosition(0.85);
-    }
+    }*/
```

Comment:
Please remove commented out code. We have Git history to restore if any of this is needed in the future.